src/java_bytecode/java_class_loader.h

```diff
@@ -51,22 +69,22 @@ class java_class_loadert:public messaget
       std::string class_file_name;
     };
 
-    // class name to index map
+    /// Maps class names to a jar file entry
```

Comment:
I'd suggest to simply rename the `entryt` class to something like `jar_file_entryt` at which point the comment becomes unnecessary (and should thus be dropped). Good code is self-documenting. (I'm not saying we overwhelmingly good code in this regard.)